libs/NfcCoreLib/inc/phLibNfc.h

```diff
@@ -62,6 +62,24 @@ typedef enum
     phLibNfc_SE_ActModeOff     = 0x01,
 }phLibNfc_eSE_ActivationMode;
 
+/**
+* \ingroup grp_nci_nfc
+* \brief NFCEE Power and Link Control Info contains power related identifier..
+*/
+typedef enum phLibNfc_PowerLinkMode
+{
+    /** NFCEE  NFCC decides identifier*/
+    phLibNfc_PLM_NfccDecides = 0x00,
+    /** NFCEE  power supply always on*/
+    phLibNfc_PLM_PowerSupplyAlwaysOn = 0x01,
+    /** NFCEE  NFCC to NFCEE Communication link always active when the NFCEE is powered on*/
+    phLibNfc_PLM_NfccComLinkActiveOnPowerOn = 0x02,
```

Comment:
Can you use the value names I recommended?

Specifically,
- *NfccDecides
- *PowerSupplyAlwaysOn
- *ComLinkActiveWhenPowerOn
- *PowerAndComLinkAlwaysOn

Reply:
I have updated the pull request following the names value. NfccDecides remains unchanged.
Hopefully no misunderstanding on my side ?